Podcast/BookmarksTableViewCell.swift

```diff
@@ -0,0 +1,228 @@
+        separator = UIView(frame: CGRect.zero)
+        separator.backgroundColor = .podcastWhiteDark
+        contentView.addSubview(separator)
+        
+        episodeImage = UIImageView(frame: CGRect.zero)
+        addSubview(episodeImage)
+        
+        episodeNameLabel = UILabel(frame: CGRect.zero)
+        episodeNameLabel.font = UIFont.boldSystemFont(ofSize: 16.0)
+        episodeNameLabel.textColor = .podcastBlack
+        addSubview(episodeNameLabel)
+        
+        dateTimeLabel = UILabel(frame: CGRect.zero)
+        dateTimeLabel.font = UIFont.systemFont(ofSize: 12.0)
+        dateTimeLabel.textColor = .podcastGrayDark
+        addSubview(dateTimeLabel)
+        
+        let labels: [UILabel] = [episodeNameLabel, dateTimeLabel]
+        for label in labels {
+            label.textAlignment = .left
+            label.lineBreakMode = .byTruncatingTail
+            label.numberOfLines = 1
+        }
+        
+        playButton = UIButton(frame: CGRect.zero)
+        playButton.setImage(#imageLiteral(resourceName: "bookmarks_play_small"), for: .normal)
+        playButton.setTitleColor(.podcastGrayDark, for: .normal)
+        playButton.setTitle("Play", for: .normal)
+        playButton.contentHorizontalAlignment = .left
+        playButton.titleEdgeInsets = UIEdgeInsets(top: 0, left: buttonTitlePadding, bottom: 0, right: 0)
+        playButton.titleLabel?.textColor = .podcastGrayDark
+        playButton.titleLabel?.font = .systemFont(ofSize: 12)
+        playButton.addTarget(self, action: #selector(didPressPlayButton), for: .touchUpInside)
+        addSubview(playButton)
+        
+        recommendedButton = UIButton(frame: CGRect.zero)
+        recommendedButton.setImage(#imageLiteral(resourceName: "bookmarks_recommend_small"), for: .normal)
+        recommendedButton.setImage(#imageLiteral(resourceName: "bookmarks_recommend_small_selected"), for: .selected)
+        recommendedButton.contentHorizontalAlignment = .left
+        recommendedButton.titleEdgeInsets = UIEdgeInsets(top: 0, left: buttonTitlePadding, bottom: 0, right: 0)
+        recommendedButton.setTitleColor(.podcastGrayDark, for: .normal)
+        recommendedButton.setTitleColor(.cancelButtonRed, for: .selected)
+        recommendedButton.setTitle("0", for: .normal)
+        recommendedButton.titleLabel?.font = .systemFont(ofSize: 12)
+        recommendedButton.addTarget(self, action: #selector(didPressRecommendedButton), for: .touchUpInside)
+        addSubview(recommendedButton)
+        
+        moreButton = UIButton(frame: CGRect.zero)
+        moreButton.setImage(#imageLiteral(resourceName: "more_icon"), for: .normal)
+        moreButton.addTarget(self, action: #selector(didPressMoreButton), for: .touchUpInside)
+        addSubview(moreButton)
+        
+        heightConstraint = NSLayoutConstraint(item: contentView, attribute: .height, relatedBy: .equal, toItem: nil, attribute: NSLayoutAttribute.notAnAttribute, multiplier: 1, constant: height)
```

Comment:
Not sure what this is, took it from CardTableViewCell, didn't think it was necessary but I didn't want to break anything, I'll remove it